lua/wire/cpulib.lua

```diff
@@ -34,6 +34,7 @@ if CLIENT then
   CPULib.Debugger.PointersByLine = {}
 
   CPULib.Debugger.Breakpoint = {}
+  CPULib.print = print
```

Comment:
Old modification, will just revert it.

Initially used to make the compiler silent by replacing the default print but I started using HCOMP's compile funcs directly instead of going through CPULib to manage the compile(CPULib is the one that throttles the compile to do (n) steps and then wait for a timer), which also makes the compile as silent and instant as we want it.

To answer the question about fenvs, I didn't know what they were until I looked them up just now, so I agree this could've been implemented better but either implementation shouldn't be necessary at the moment.